### PR TITLE
[DOCS-1054] Prefer "W&B" over "Weights & Biases" in most situations

### PR DESCRIPTION
**Pull request by mdlinville**
[DOCS-1054] Prefer "W&B" over "Weights & Biases" in most situations

- Intentionally did not update the home page
- Updated Guides,Support, Tutorials, non-generated references, and Launch
- In lines I changed, I also removed emojis and trailing space and made sure sentences were complete.
- The JIRA requested that we use a variable for the company name, but I skipped that since we are probably moving off Hugo.

## Previews
Too many to list individually.

## Tests
- [x] Local build
- [x] [Link check](https://github.com/wandb/docs/actions/runs/16735478563)  (5 errors, none related to this PR)

[DOCS-1054]: https://wandb.atlassian.net/browse/DOCS-1054?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/guides/core/automations/create-automations/slack.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ A team admin can add a Slack integration to the team.
 
     ![Screenshot showing two Slack integrations in a Team](/images/automations/slack_integrations.png)
 1. If necessary, sign in to Slack in your browser. When prompted, grant W&B permission to post to the Slack channel you select. Read the page, then click **Search for a channel** and begin typing the channel name. Select the channel from the list, then click **Allow**.
-1. In Slack, go to the channel you selected. If you see a post like `[Your Slack handle] added an integration to this channel: Weights & Biases`, the integration is configured correctly.
+1. In Slack, go to the channel you selected. If you see a post like `[Your Slack handle] added an integration to this channel: W&B`, the integration is configured correctly.
```
Comment (mdlinville) [inline content/en/guides/core/automations/create-automations/slack.md]:
```suggestion
1. In Slack, go to the channel you selected. If you see a post like `[Your Slack handle] added an integration to this channel: Weights & Biases`, the integration is configured correctly.
```

File: content/en/guides/integrations/diffusers.md [hunk 2]
```diff
@@ -43,11 +43,11 @@ autolog(init=dict(project="diffusers_logging"))
         ```
 
 
-2. Use `autolog` to initialize a Weights & Biases run and automatically track the inputs and the outputs from [all supported pipeline calls](https://github.com/wandb/wandb/blob/main/wandb/integration/diffusers/autologger.py#L12-L72).
+2. Use `autolog` to initialize a W&B run and automatically track the inputs and the outputs from [all supported pipeline calls](https://github.com/wandb/wandb/blob/main/wandb/integration/diffusers/autologger.py#L12-L72).
```
Comment (mdlinville) [inline content/en/guides/integrations/diffusers.md]:
```suggestion
2. Use `autolog` to initialize a W&B Run and automatically track the inputs and the outputs from [all supported pipeline calls](https://github.com/wandb/wandb/blob/main/wandb/integration/diffusers/autologger.py#L12-L72).
```
At the first mention of a W&B-specific functionality like "W&B Run", the word after W&B should be capitalized. Afterward, you don't need to use W&B and the word should be lowercased like "run". Same thing for W&B Registry, W&B Sweeps, W&B Artifact, etc.

File: content/en/guides/integrations/lightgbm.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Looking for working code examples? Check out [our repository of examples on GitH
 
 ## Tuning your hyperparameters with Sweeps
 
-Attaining the maximum performance out of models requires tuning hyperparameters, like tree depth and learning rate. Weights & Biases includes [Sweeps]({{< relref "/guides/models/sweeps/" >}}), a powerful toolkit for configuring, orchestrating, and analyzing large hyperparameter testing experiments.
+Attaining the maximum performance out of models requires tuning hyperparameters, like tree depth and learning rate. W&B includes [Sweeps]({{< relref "/guides/models/sweeps/" >}}), a powerful toolkit for configuring, orchestrating, and analyzing large hyperparameter testing experiments.
```
Comment (mdlinville) [inline content/en/guides/integrations/lightgbm.md]:
```suggestion
Attaining the maximum performance out of models requires tuning hyperparameters, like tree depth and learning rate. W&B [Sweeps]({{< relref "/guides/models/sweeps/" >}}) is a powerful toolkit for configuring, orchestrating, and analyzing large hyperparameter testing experiments.
```
Where we see a pattern like this, we should simplify it.

File: content/en/guides/integrations/lightning.md [hunk 4]
```diff
@@ -512,7 +512,7 @@ wandb_logger.log_table(key="my_samples", columns=columns, data=data)
 
 {{< /tabpane >}}
 
-You can use Lightning's Callbacks system to control when you log to Weights & Biases via the WandbLogger, in this example we log a sample of our validation images and predictions:
+You can use Lightning's Callbacks system to control when you log to W&B via the WandbLogger, in this example we log a sample of our validation images and predictions:
```
Comment (mdlinville) [inline content/en/guides/integrations/lightning.md]: `WandbLogger` should always be marked up in code font.

File: content/en/guides/integrations/skorch.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ title: Skorch
 weight: 400
 ---
 
-You can use Weights & Biases with Skorch to automatically log the model with the best performance, along with all model performance metrics, the model topology and compute resources after each epoch. Every file saved in `wandb_run.dir` is automatically logged to W&B servers.
+You can use W&B with Skorch to automatically log the model with the best performance, along with all model performance metrics, the model topology and compute resources after each epoch. Every file saved in `wandb_run.dir` is automatically logged to W&B servers.
```
Comment (mdlinville) [inline content/en/guides/integrations/skorch.md]:
```suggestion
You can use W&B with Skorch to automatically log the model with the best performance, along with all model performance metrics, the model topology and compute resources after each epoch. Every file saved in `wandb_run.dir` is automatically logged to W&B.
```
Most of the time, we don't refer to "W&B servers". Usually, we just use "W&B". If we are specifically talking about a self-hosted instance of W&B, we instead refer to "W&B Dedicated Cloud" or "W&B Self-Managed".

File: content/en/guides/integrations/spacy.md [hunk 6]
```diff
@@ -138,4 +138,4 @@ python -m spacy train \
 {{% /tab %}}
 {{< /tabpane >}}
 
-When training begins, a link to your training run's [W&B page]({{< relref "/guides/models/track/runs/" >}}) will be output which will take you to this run's experiment tracking [dashboard]({{< relref "/guides/models/track/workspaces.md" >}}) in the Weights & Biases web UI.
\ No newline at end of file
+When training begins, a link to your training run's [W&B page]({{< relref "/guides/models/track/runs/" >}}) will be output which will take you to this run's experiment tracking [dashboard]({{< relref "/guides/models/track/workspaces.md" >}}) in the W&B web UI.
```
Comment (mdlinville) [inline content/en/guides/integrations/spacy.md:141]:
```suggestion
When training begins, a link to your training run's [W&B page]({{< relref "/guides/models/track/runs/" >}}) will be output which will take you to this run's experiment tracking [dashboard]({{< relref "/guides/models/track/workspaces.md" >}}) in the W&B AppI.
```
We prefer "W&B App" over "W&B web UI", "W&B App UI", or other variants.

File: content/en/guides/integrations/xgboost.md [hunk 7]
```diff
@@ -29,22 +29,22 @@ with wandb.init() as run:
   bst.fit(X_train, y_train, callbacks=[WandbCallback(log_model=True)])
 ```
 
-You can open [this notebook](https://wandb.me/xgboost) for a comprehensive look at logging with XGBoost and Weights & Biases
+You can open [this notebook](https://wandb.me/xgboost) for a comprehensive look at logging with XGBoost and W&B
```
Comment (mdlinville) [inline content/en/guides/integrations/xgboost.md:32]: Incomplete sentence

File: content/en/guides/integrations/xgboost.md [hunk 8]
```diff
@@ -29,22 +29,22 @@ with wandb.init() as run:
...
-- log the booster model configuration to Weights & Biases
-- log evaluation metrics collected by XGBoost, such as rmse, accuracy etc to Weights & Biases
+- log the booster model configuration to W&B
+- log evaluation metrics collected by XGBoost, such as rmse, accuracy etc to W&B
 - log training metrics collected by XGBoost (if you provide data to eval_set)
 - log the best score and the best iteration
-- save and upload your trained model to Weights & Biases Artifacts (when `log_model = True`)
+- save and upload your trained model to W&B Artifacts (when `log_model = True`)
 - log feature importance plot when `log_feature_importance=True` (default).
 - Capture the best eval metric in `wandb.Run.summary` when `define_metric=True` (default).
```
Comment (mdlinville) [inline content/en/guides/integrations/xgboost.md:44]: These should start with capital letters and end in periods.

File: content/en/guides/integrations/xgboost.md [hunk 9]
```diff
@@ -29,22 +29,22 @@ with wandb.init() as run:
...
 
 ### Arguments
-- `log_model`: (boolean) if True save and upload the model to Weights & Biases Artifacts
+- `log_model`: (boolean) if True save and upload the model to W&B Artifacts
 
 - `log_feature_importance`: (boolean) if True log a feature importance bar plot
```
Comment (mdlinville) [inline content/en/guides/integrations/xgboost.md:49]: These should start with capital letters and end in periods (through line 53). The pattern is that list items that are longer than a few words should be complete sentences.

File: content/en/launch/integration-guides/dagster.md [hunk 10]
```diff
@@ -756,7 +756,7 @@ def read_specific_partitions(context, my_daily_partitioned_asset):
 {{% /tab %}}
 {{< /tabpane >}}
 
-The configuration object, `metadata`, is used to configure how Weights & Biases (wandb) interacts with different artifact partitions in your project.
+The configuration object, `metadata`, is used to configure how W&B (wandb) interacts with different artifact partitions in your project.
```
Comment (mdlinville) [inline content/en/launch/integration-guides/dagster.md]:
```suggestion
The configuration object, `metadata`, configures how W&B interacts with different artifact partitions in your project.
```
This pattern of simplification may also be relevant to other changed lines.

File: content/en/launch/integration-guides/minikube_gpu.md [hunk 11]
```diff
@@ -20,7 +20,7 @@ W&B recommends you use a [Launch Docker queue]({{< relref "/launch/set-up-launch
...
+The [Nvidia container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/index.html) has made it easy to run GPU-enabled workflows on Docker. One limitation is a lack of native support for scheduling GPU by volume. If you want to use a GPU with the `docker run` command you must either request specific GPU by ID or all GPU present, which makes many distributed GPU enabled workloads impractical. Kubernetes offers support for scheduling by a volume request, but setting up a local Kubernetes cluster with GPU scheduling can take considerable time and effort, until recently. Minikube, one of the most popular tools for running single node Kubernetes clusters, recently released [support for GPU scheduling](https://minikube.sigs.k8s.io/docs/tutorials/nvidia/). In this tutorial, we will create a Minikube cluster on a multi-GPU machine and launch concurrent stable diffusion inference jobs to the cluster using W&B Launch.
```
Comment (mdlinville) [inline content/en/launch/integration-guides/minikube_gpu.md]: Use NVIDIA, to align with https://www.nvidia.com/en-us/about-nvidia/legal-info/logo-brand-usage/.

File: content/en/tutorials/artifacts.md [hunk 12]
```diff
@@ -189,20 +189,20 @@ To log something as an `Artifact`, we have to first make an `Artifact` object.
 
 Every `Artifact` has a `name` -- that's what the first argument sets.
 
-> **Rule of 👍**: the `name` should be descriptive, but easy to remember and type --
+> **Best practice**: the `name` should be descriptive, but easy to remember and type --
```
Comment (mdlinville) [inline content/en/tutorials/artifacts.md]: I think the two hyphens at the end are incorrect. It should probably be a period.

File: content/en/tutorials/integration-tutorials/lightning.md [hunk 13]
```diff
@@ -318,4 +318,4 @@ I hope you find this report helpful. I will encourage to play with the code and
 
 Here are some resources to learn more about PyTorch Lightning:
 - [Step-by-step walk-through](https://lightning.ai/docs/pytorch/latest/starter/introduction.html) - This is one of the official tutorials. Their documentation is really well written and I highly encourage it as a good learning resource.
-- [Use Pytorch Lightning with Weights & Biases](https://wandb.me/lightning) - This is a quick colab that you can run through to learn more about how to use W&B with PyTorch Lightning.
\ No newline at end of file
+- [Use Pytorch Lightning with W&B](https://wandb.me/lightning) - This is a quick colab that you can run through to learn more about how to use W&B with PyTorch Lightning.
```
Comment (mdlinville) [inline content/en/tutorials/integration-tutorials/lightning.md]: Within a list item, use colons instead of dashes to separate a summary from more info.

File: content/en/tutorials/integration-tutorials/pytorch.md [hunk 14]
```diff
@@ -8,13 +8,13 @@ weight: 1
 ---
 {{< cta-button colabLink="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/pytorch/Simple_PyTorch_Integration.ipynb" >}}
 
-Use [Weights & Biases](https://wandb.com) for machine learning experiment tracking, dataset versioning, and project collaboration.
+Use [W&B](https://wandb.com) for machine learning experiment tracking, dataset versioning, and project collaboration.
```
Comment (mdlinville) [inline content/en/tutorials/integration-tutorials/pytorch.md]: Prefer https://wandb.ai.

File: content/en/guides/integrations/accelerate.md [hunk 15]
```diff
@@ -79,10 +79,10 @@ Below is an Accelerate article you may enjoy
 
 <details>
 
-<summary>HuggingFace Accelerate Super Charged With Weights & Biases</summary>
+<summary>HuggingFace Accelerate Super Charged With W&B</summary>
 
-* In this article, we'll look at what HuggingFace Accelerate has to offer and how simple it is to perform distributed training and evaluation, while logging results to Weights & Biases
+* In this article, we'll look at what HuggingFace Accelerate has to offer and how simple it is to perform distributed training and evaluation, while logging results to W&B
```
Comment (mdlinville) [inline content/en/guides/integrations/accelerate.md]: This sentence is missing punctuation.

File: content/en/guides/integrations/deepchecks.md [hunk 16]
```diff
@@ -32,7 +32,7 @@ result = ModelErrorAnalysis()
 result.to_wandb()
 ```
 
-You can also log an entire DeepChecks test suite to Weights & Biases
+You can also log an entire DeepChecks test suite to W&B
```
Comment (mdlinville) [inline content/en/guides/integrations/deepchecks.md]: This is missing punctuation.
